src/odemis/util/test.py

```diff
@@ -136,7 +136,15 @@ def stop_backend():
     model._core._microscope = None  # force reset of the microscope for next connection
 
     if status != driver.BACKEND_STOPPED:
-        raise IOError("Backend failed to stop, now %s" % status)
+        logging.warning("Backend failed to stop, will run odemis-stop to kill it.")
```

Comment:
I think it is useful to mention the fact that you use sudo for this call.
logging.warning("Backend failed to stop, will run 'sudo odemis-stop' to kill it.")